internal/pkg/pct/mock_utils_test.go

```diff
@@ -0,0 +1,103 @@
+package pct
```

Comment:
I was in two minds about suggesting this, but if we're consolidating everything in to one file, maybe some comments to act as a delineator between each mock object's function overrides, just to make it a bit easier to spot at a glance, where things live:

```go
// osUtils
func (osUtilHelpersImplMock) IsNotExist(err error) bool {
...

// gzipUtils
func (gzipHelpersImplMock) Gzip(source, target string) (gzipFilePath string, err error) {
...
```